src/Parser/Core/Modules/ManaValues.js

```diff
@@ -0,0 +1,34 @@
+import Module from 'Parser/Core/Module';
+
+const debug = false;
+
+const MANA_CLASS_RESOURCE_ID = 0;
+
+class ManaValue extends Module {
+  lowestMana = 1100000;
+  endingMana = 0;
+
+  on_byPlayer_cast(event) {
+    // class resource type 0 means the resource is mana
+    if (event.classResources && event.classResources[0] && event.classResources[0].type === MANA_CLASS_RESOURCE_ID) {
```

Comment:
I am uncertain mana is always the first resource. I think it would be more reliable to just loop over the classResources instead of relying on it always being the first entry, like so:

```js
event.classResources.forEach(classResource => { 
  ... 
});
```